### PR TITLE
Scto guidelines

### PR DESCRIPTION
**Pull request by kbjarkefur**
Add step by step guidelines for encryption in SurveyCTO

### CODE REVIEW

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+Then click _Download .csv now_.
+
+<img src="https://github.com/worldbank/dime-standards/blob/scto-guidelines/dime-research-standards/pillar-4-data-security/data-security-resources/img/scto-publish-1.png" width="75%"><!--- Image is read from master branch or use full URL-->
+
+## Topic 4 - Using your private key to decrypt your data in SurveyCTO
+
+Technically you can download encrypted data from your SurveyCTO server using a browser,
+but then you first need to save your private key in a file on your computer.
+In the workflow we recommend,
+you really want to avoid to ever store the private key in a file on your computer
```
Comment (bbdaniels) [inline dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md:224]:
```suggestion
you avoid ever storing the private key in a file on your computer
```

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+after you have saved it in a password manager.
+Instead, we recommend you to only download data using [SurveyCTO Desktop](https://docs.surveycto.com/desktop/)
```
Comment (bbdaniels) [inline dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md:226]:
```suggestion
Instead, we recommend to only download data using [SurveyCTO Desktop](https://docs.surveycto.com/desktop/)
```

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+and click _Sync_ in the menu to the left.
+Fill in the information as normal,
+and in step 2 you need to paste the private key. See image below.
+Go to your password manager and copy your private key.
+Remember to include the header `-----BEGIN RSA PRIVATE KEY-----`
+and the footer `-----END RSA PRIVATE KEY-----`.
+Once you have copied it to your clipboard
+(ctrl-C on Windows and command-C on Mac)
+then simply click the _PASTE KEY_ button in SurveyCTO Desktop.
+If the key you pasted is on the excepted format,
```
Comment (bbdaniels) [inline dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md:240]:
```suggestion
If the key you pasted is on the expected format,
```

Maybe a typo for "accepted"? But I think this is better anyway

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+then you will see a green checkmark
+and the text "_Private key successfully pasted_".
+However, SurveyCTO has not tested if you pasted the correct key,
+it will only test that when you are actually downloading data.
+If you have pasted the incorrect key, then you will get an error
+and you will not be able to read the data.
+
+<img src="https://github.com/worldbank/dime-standards/blob/scto-guidelines/dime-research-standards/pillar-4-data-security/data-security-resources/img/scto-sync-1.png" width="75%"><!--- Image is read from master branch or use full URL-->
+
+## Topic 5 - Testing your setup before
```
Comment (bbdaniels) [inline dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md:250]: before what???? ;-)

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+
+Testing your encryption and decryption workflow is easy
+but we cannot stress enough how important it is that
+you do indeed test the workflow before you start to collect real data.
+
+The first thing you want to make sure that your form is indeed encrypted.
+The easiest way to do that is to log in to your server and go to the _Design_ tab.
+Find your survey and see if the icon for encryption is an open or closed pad lock.
```
Comment (bbdaniels) [inline dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md:258]:
```suggestion
Find your survey and see if the icon for encryption is an open or closed padlock.
```

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+See the examples below where the form named _encrypted_form_ has a closed padlock
+and the form named _unencrypted_form_ has an open padlock.
+IF the padlock is closed then you know that all fields that
+are **not** listed as publishable will be encrypted during data collection.
+
+<img src="https://github.com/worldbank/dime-standards/blob/scto-guidelines/dime-research-standards/pillar-4-data-security/data-security-resources/img/scto-test-1.png" width="100%"><!--- Image is read from master branch or use full URL-->
+
+However, there is a second things you really want to test,
```
Comment (bbdaniels) [inline dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md:266]:
```suggestion
However, there is a second thing you really want to test,
```

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md [hunk 7]
```diff
@@ -0,0 +1,274 @@
...
+and that is to make sure you are able to decrypt the encrypted data.
+To test your that you can decrypt data,
+simply submit one mock data submission,
```
Comment (bbdaniels) [inline dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md:269]:
```suggestion
simply submit one trial data submission,
```

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md [hunk 8]
```diff
@@ -0,0 +1,274 @@
...
+then copy the private key from your password manager,
+and use SurveyCTO Desktop to download your mock submission.
```
Comment (bbdaniels) [inline dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md:271]:
```suggestion
and use SurveyCTO Desktop to download your trial submission.
```

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md [hunk 9]
```diff
@@ -0,0 +1,275 @@
...
+Each public/private key pair is unique and no other private key can decrypt the data you encrypted with your public key.
+You should never use the same key pair for two different projects,
+but you do not need to create a new key pair for every form within a project.
+We think that a good balance is to use one key pair for each project, and this practice typically conforms with IRB rules for regulating data access at the project level.
+
+
+As the name suggests, it is essential to keep your private key secret. They key provides access to your encrypted data.
+If you accidentally publish your private key or store it in an insecure location, then your data is no longer securely protected.
+If you lose access to your private key, then there is no way to decrypt your data -- neither the software nor the service provider can restore it.
+
```
Comment (kbjarkefur) [inline dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md:47]:
```suggestion
```

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md [hunk 10]
```diff
@@ -0,0 +1,274 @@
...
+Upload the form you with the public key included in the settings tab,
+and then follow the instructions as normal.
+SurveyCTO will test that there are no errors in the public key,
+but you should never start collecting data using an encrypted form before
+you have followed our test instructions below.
+
+## Topic 3 - Publishable fields
+
+One often overseen feature when encrypting forms are publishable fields.
```
[the remark this answers is not in the document]
Comment (kbjarkefur) [reply dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md:184]: haha, this is funny! Thanks!

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md (hunk 4 above)
Comment (kbjarkefur) [reply dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md:247]: done in efac4632a617c269d6083b8a7ffa1a2ff104ea00

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md [hunk 11]
```diff
@@ -0,0 +1,274 @@
...
+We will provide instructions for the password manager LastPass,
+but this can be done in other password managers too.
+A secure alternative to saving to storing the keys in a password manager
+is to store the keys in an [encrypted folder](https://github.com/worldbank/dime-standards/blob/scto-guidelines/dime-research-standards/pillar-4-data-security/data-security-resources/veracrypt-guidelines.mdt)
+on your computer,
+but then you still need to store the key to the encrypted folder in a password manager.
+
+Go to lastpass.com, log in to your vault and
+click the plus sign in the red circle to create a new item.
+Select _Secure Note_
```
[the remark this answers is not in the document]
Comment (kbjarkefur) [reply dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md]: done in efac4632a617c269d6083b8a7ffa1a2ff104ea00

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md [hunk 12]
```diff
@@ -0,0 +1,274 @@
...
+
+<img src="https://github.com/worldbank/dime-standards/blob/scto-guidelines/dime-research-standards/pillar-4-data-security/data-security-resources/img/scto-encrypt-store-1.png" width="50%"><!--- Image is read from master branch or use full URL-->
+
+Then copy all the content of both keys you created and downloaded from your SurveyCTO server.
```
[the remark this answers is not in the document]
Comment (kbjarkefur) [reply dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md:105]: done in efac4632a617c269d6083b8a7ffa1a2ff104ea00

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md [hunk 13]
```diff
@@ -0,0 +1,274 @@
...
+
+
+### Step 1.1 - Creating a public/private key pair.
+
+You can create the public/private key pair on your SurveyCTO server.
+Log in to your server, and navigate to the _Design_ tab. Then click _Tools_.
+
+<img src="https://github.com/worldbank/dime-standards/blob/scto-guidelines/dime-research-standards/pillar-4-data-security/data-security-resources/img/scto-encrypt-create-1.png" width="75%"><!--- Image is read from master branch or use full URL-->
+
+Then click _create a new key_ and then _Start key generator_
```
[the remark this answers is not in the document]
Comment (kbjarkefur) [reply dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md]: done in efac4632a617c269d6083b8a7ffa1a2ff104ea00

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (kbjarkefur) [reply dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md]: done in e6f8855dd567faf20c5955f41f8ced7e3486abaf

File: dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md (hunk 4 above)
Comment (kbjarkefur) [reply dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md]: done in efac4632a617c269d6083b8a7ffa1a2ff104ea00

**Pull request merged**